deformable_manipulation_experiment_params/launch/cloth_single_pole_params.launch

```diff
@@ -83,16 +89,23 @@
         <param name="viewer_height"                     value="800"     type="int"/>
     </group>
 
+    <!-- planner, controller params -->
+    <arg name="stretching_cosine_threshold"    default="0.55"/>
+    <arg name="desired_down_scale"             default="400"/>
+    <arg name="fix_step_size"                  default="false"/>
+
     <group ns="smmap_planner_node">
         <group ns="task">
             <param name="max_time"                              value="$(arg task_max_time)"    type="double"/>
             <param name="error_threshold_along_normal"          value="0.02"                    type="double"/>
             <param name="error_threshold_distance_to_normal"    value="0.006"                   type="double"/>
             <param name="error_threshold_task_done"             value="0.1"                     type="double"/>
+            <!-- <param name="max_stretch_factor"            value="$(eval (0.012 + 0.002) / 0.012)" type="double"/> -->
```

Comment:
Same here, only 1 value for these parameters in the master branch. I'm not going to annotate every instance at this point.